HISTORY.md

```diff
@@ -1,6 +1,14 @@
-### mlpack 3.1.0
+### mlpack 3.1.1
 ###### ????-??-??
+  * `output` option changed to `predictions` for adaboost and perceptron
+    binding.Old options are now deprecated and will be preserved until mlpack
```

Comment:
space after `.` , I'll make that changes after , your comments about the Line no !

Reply:
Added Space